src/Components/Web/test/HtmlRendering/HtmlRendererTest.cs

```diff
@@ -649,21 +642,19 @@ public async Task RenderComponentAsync_CanPassParameters()
     {
         // Arrange
         var expectedHtml = new[] {
-                "<", "p", ">", "<", "input", " ", "value", "=", "\"", "5", "\"", " />", "</", "p", ">" };
+                "<", "p", ">", "<", "input", " ", "configure", "=", "\"", "5", "\"", " />", "</", "p", ">" };
 
         RenderFragment Content(ParameterView pc) => new RenderFragment((RenderTreeBuilder rtb) =>
         {
             rtb.OpenElement(0, "p");
             rtb.OpenElement(1, "input");
             rtb.AddAttribute(2, "change", pc.GetValueOrDefault<Action<ChangeEventArgs>>("update"));
-            rtb.AddAttribute(3, "value", pc.GetValueOrDefault<int>("value"));
+            rtb.AddAttribute(3, "configure", pc.GetValueOrDefault<int>("configure"));
```

Comment:
Out of interest, is there some reason this attribute name needed to change? Just want to check there isn't some actual product behavioral change this represents.

Reply:
No, it's a bug in the find/replace stuff. I thought I had "cleaned" up all the places it messed up. I'll fix it